core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -2493,8 +2506,8 @@ showWalletDataWindow.walletData=Wallet data
 showWalletDataWindow.includePrivKeys=Include private keys
 
 setXMRTxKeyWindow.headline=Prove sending of XMR
-setXMRTxKeyWindow.txHash=Transaction hash
-setXMRTxKeyWindow.txKey=Tx private key
+setXMRTxKeyWindow.txHash=Transaction id
+setXMRTxKeyWindow.txKey=Tx key
```

Comment:
How about "Transaction key" instead?  The term "private key" always raises red flags because we are conditioned to never reveal our private keys.  

- https://www.getmonero.org/resources/user-guides/prove-payment.html refers to it as "the transaction's key".  Their web form calls it a "secret transaction key", slightly less intimidating without the word private.
- monero-wallet-cli also calls it tx key: `get_tx_key TXID`
- https://monero.stackexchange.com/questions/1860/how-do-i-prove-that-a-payment-has-been-sent-to-a-particular-address/1861 they call it transaction key.
- https://github.com/moneroexamples/onion-monero-blockchain-explorer calls it `viewkey`.
- Bisq's contract text for XMR account creation calls it `Transaction key` three times, and also gives synonyms `Tx Key, Tx Secret Key or Tx Private Key`
- https://xmr.llcoins.net/checktx.html does use "private key" though.